# OpenCV_Recognizer

TESTED on UBUNTU 19.04

You may have to change some path parameters on Windows Systems!

USAGE:

-execute create_datasets.py with an argument that specifies the label of the dataset
(this will create the dataset in resources/faces_2_recognize)
-execute train_faces.py (this will generate data inside recognizer folder)
(YOU WILL NEED AT LEAST 2 DATASETS IN faces_2_recognize FOLDER IN ORDER FOR THIS TO WORK!)
-execute detect_faces.py 


:D
